adapters/pixfuture/params_test.go

```diff
@@ -0,0 +1,41 @@
+package pixfuture
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	require.NoError(t, err, "Failed to fetch the json schema.")
+
+	for _, p := range validParams {
+		assert.NoErrorf(t, validator.Validate(openrtb_ext.BidderPixfuture, json.RawMessage(p)), "Schema rejected valid params: %s", p)
+	}
+}
```

Comment:
The pattern that is used quite often is to create a list of tests and to t.Run each - that way each test is counted individually in any reports, and can be run in IDE directly / individually since the test can be referenced with a pattern. Given what you have below, something like this would be good:

```suggestion
func TestValidParams(t *testing.T) {
	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
	require.NoError(t, err, "Failed to fetch the json schema.")

	tests := struct{
		name string
		json string
	}{
		{
			name: "Minimum length satisfied",
			json: `{"pix_id": "123"}`,
		},
		{
			name: "Longer valid string",
			json: `{"pix_id": "abcdef"}`,
		},
	}
	for _, tt := range tests {
		t.Run(tt.name, func() {
		assert.NoErrorf(t, validator.Validate(openrtb_ext.BidderPixfuture, json.RawMessage(tt.json)), "Schema rejected valid params: %s", p)
		})
	}
}
```

Reply:
The recommended code refactoring has been completed.